lambda-notes/src/Components/leftContent/Nav.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Nav = () => {
+	return (
+		<div>
+			<Link to={'/'}>Lambda Notes</Link>
+			<Link to={'/'}>View Your Notes</Link>
+			<Link to={'/'}>+Creat New Note</Link>
```

Comment:
The form for creating and editing notes should be a separate page, but I notice you haven't implemented those routes yet. Just don't forget to update your Links to reflect the proper routes.

Reply:
These paths don't match your Routes yet.